charts/netdata/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: netdata
-version: 3.6.13
+version: 3.6.14
```

Comment:
[Version badge](https://github.com/netdata/helmchart/tree/master/charts/netdata#netdata-helm-chart-for-kubernetes-deployments) needs to be updated too